portfolio/pom.xml

```
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+  <dependency>
```

```suggestion
      <dependency>
```
Please fix the indentation so that the `<dependency></dependency>` tags are aligned

---

I missed that, thank you.